DirectEmployers/MyJobs
# Order and limit communication records [PD-3887]

## What's this PR do?
Orders returned communication records by `date_time` and allows limiting the returned records with `&limit=&offset=`.
## Screenshots (if appropriate)
N/A
## Where should the reviewer start?
mypartners/views.py
## How should this be manually tested?
Open `/prm/api/communicationrecords/?partner_id=<partner_id>` and ensure things are ordered and nothing is limited.
Tack on some combination of `&limit=&offset=` and ensure things are limited in the manner you would expect.
## Any background context you want to provide?
This will be used for the dashboard system we're working on. @aljordan82 thought others might want it.

This is builtin rest_framework functionality. I'd prefer not writing tests for external code, hence why you don't see any here.
## PR Checklist

### Separation of Concerns


### Tests
